frontend/src/components/placementVisuals/Placement.tsx

```diff
@@ -144,37 +135,12 @@ function Placement({ stepID, sampleIDs }: PlacementProps) {
 
     const transferAllSamples = useCallback(() => {
         if (activeSourceContainer && activeDestinationContainer)
-            dispatch(placeAllSource({ source: activeSourceContainer.name, destination: activeDestinationContainer.name }))
+            dispatch(placeAllSource({ source: activeSourceContainer, destination: activeDestinationContainer }))
     }, [activeDestinationContainer, activeSourceContainer, dispatch])
 
-    const clearSelection = useCallback(() => {
-        batch(() => {
-            for (const parentContainer of sourceContainers) {
-                dispatch(multiSelect({
-                    parentContainer: parentContainer.name,
-                    type: 'all',
-                    forcedSelectedValue: false,
-                    context: {
-                        source: activeSourceContainer?.name
-                    }
-                }))
-            }
-            for (const parentContainer of destinationContainers) {
-                dispatch(multiSelect({
-                    parentContainer: parentContainer.name,
-                    type: 'all',
-                    forcedSelectedValue: false,
-                    context: {
-                        source: activeSourceContainer?.name
-                    }
-                }))
-            }
-        })
-    }, [activeSourceContainer?.name, destinationContainers, dispatch, sourceContainers])
-
-    const removeSelectedCells = useCallback(() => {
+    const undoPlacementsCallback = useCallback(() => {
         if (activeDestinationContainer) {
-            dispatch(undoSelectedSamples(activeDestinationContainer.name))
+            dispatch(undoPlacements(activeDestinationContainer.name))
```

Comment:
We pass the container for the placeAllSource we should do the same for the undoPlacements.

Reply:
Huh? But I am passing the container `activeDestinationContainer.name`

Reply:
It is just a question of consistency :) Up to you.

Reply:
Where's the inconsistency? 🤔